mava/wrappers/smac.py

```diff
@@ -281,7 +284,24 @@ def get_stats(self) -> Optional[Dict]:
         Returns:
             extra stats to be logged.
         """
-        return self._environment.get_stats()
+        stats = self._environment.get_stats()
+        stats["cumulative_win_rate"] = stats["win_rate"]
+        del stats["win_rate"]
+        return stats
+
+    def get_extra_stats(self) -> Optional[Dict]:
+        """Return extra stats to be logged.
```

Comment:
Also, please change this docstring as it has the same description as `get_stats` above

Reply:
Okay!

Reply:
Done!